02-getting-started/projects/challenge/iosApp/iosApp/ContentView.swift

```diff
@@ -2,8 +2,10 @@ import SwiftUI
 import shared
 
 struct ContentView: View {
+	let greet = Greeting().greeting()
```

Comment:
The reader is expected to delete the `Greeting` class as part of the chapter. So this class won't be in the final project and hence the challenge project

Reply:
Yes